internal/dag/httpproxy_processor.go

```diff
@@ -271,16 +270,15 @@ func (p *HTTPProxyProcessor) computeHTTPProxy(proxy *contour_api_v1.HTTPProxy) {
 					return
 				}
 
-				sec, err = p.source.LookupTLSSecret(*p.FallbackCertificate)
+				sec, err = p.source.LookupTLSSecret(*p.FallbackCertificate, proxy.Namespace)
 				if err != nil {
-					validCond.AddErrorf(contour_api_v1.ConditionTypeTLSError, "FallbackNotValid",
-						"Spec.Virtualhost.TLS Secret %q fallback certificate is invalid: %s", p.FallbackCertificate, err)
-					return
-				}
-
-				if !p.source.DelegationPermitted(*p.FallbackCertificate, proxy.Namespace) {
-					validCond.AddErrorf(contour_api_v1.ConditionTypeTLSError, "FallbackNotDelegated",
-						"Spec.VirtualHost.TLS fallback Secret %q is not configured for certificate delegation", p.FallbackCertificate)
+					if _, ok := err.(DelegationNotPermittedError); ok {
+						validCond.AddErrorf(contour_api_v1.ConditionTypeTLSError, "FallbackNotDelegated",
+							"Spec.VirtualHost.TLS Secret %q is not configured for certificate delegation", p.FallbackCertificate)
+					} else {
+						validCond.AddErrorf(contour_api_v1.ConditionTypeTLSError, "FallbackNotValid",
+							"Spec.Virtualhost.TLS Secret %q fallback certificate is invalid: %s", p.FallbackCertificate, err)
+					}
```

Comment:
Handling of fallback certificate is bit different than other globally configured ones: it requires delegation to be set up ([doc link](https://projectcontour.io/docs/1.25/config/tls-termination/#fallback-certificate)). 

This could be loosened since fallback certificate reference is configured by administrator - not by namespaced tenant. Envoy client certificate is similar case and we do not require delegation there. There are also other examples, like extension service names for rate limiting service and and tracing that also accept cross-namespace names from admin without delegation resource.